### DISCUSSION TITLE
参考文献问题

### DISCUSSION THREAD

**Discussion by yemo0399** [note286/xduts#98] [Q&A] [answered]
求助！修改bib文件后，参考文献没有变，文中的上角标没有了，ctrl+点击不能跳转
使用参考文献方式和参考文献路径如下：

<img width="106" alt="微信图片_20230324205604" src="https://user-images.githubusercontent.com/128452297/227527319-cb9ac3c8-6218-43c5-9920-59c45edba47c.png">
<img width="143" alt="微信图片_20230324205538" src="https://user-images.githubusercontent.com/128452297/227527340-24266298-b663-457c-bc0e-73f677a106da.png">

**Comment by note286** [comment, marked as answer]
需要四次编译：

```shell
xelatex -synctex=-1 main
bibtex main
xelatex -synctex=-1 main
xelatex -synctex=-1 main
```

**Reply by yemo0399** [reply]
在控制台吗，需要输入文件名，我太笨了，有点不太明白
<img width="544" alt="微信图片_20230324210247" src="https://user-images.githubusercontent.com/128452297/227528440-0c633cc5-8952-416f-94c0-6ae9153918a5.png">

**Reply by note286** [reply]
我无法复现你的问题，我只是告诉你，参考文献需要这么编译而已，你可以选择使用你的编辑器提供的编译按钮来完成，但是我无法指导你如何点击以及点击哪个按钮。

**Reply by yemo0399** [reply]
您发的这个四次编译的命令中的main具体指哪个文件呀，我在控制台运行后提示没有main文件

**Reply by note286** [reply]
`main.tex`

**Reply by note286** [reply]
你可以去看一下[《一份（不太）简短的LaTeX2ε介绍》](http://mirrors.ctan.org/info/lshort/chinese/lshort-zh-cn.pdf)第6.1.4节。

**Reply by yemo0399** [reply]
十分感谢大佬，问题应该解决了
